rosdep/python.yaml

```
@@ -173,6 +173,10 @@ cython3:
   gentoo: [dev-python/cython]
   nixos: [python3Packages.cython]
   ubuntu: [cython3]
+dm-env-pip:
+  ubuntu:
+    pip:
+      packages: [dm-env]
```

Actually, sorry again.  A couple of things here:

1.  Since this is a pip-package, it should be `python3-dm-env-pip`.
2.  We haven't added this to the CONTRIBUTING.md yet, but we want to switch to wildcards for pip packages going forward.

So this whole thing would be:

```suggestion
python3-dm-env-pip:
  '*':
    pip:
      packages: [dm-env]
```

(don't forget to realphabetize once you change the key name)

---

[ef51404](https://github.com/ros/rosdistro/pull/40747/commits/ef514049c134e621ebfad68b320926cfb1a45fa4)